docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
+---
+
+# otelcol.exporter.loadbalancing
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`otelcol.exporter.loadbalancing` accepts logs and/or traces from other `otelcol` components
+and writes them over the network using the OpenTelemetry Protocol (OTLP) protocol. 
+
+> **NOTE**: `otelcol.exporter.loadbalancing` is a wrapper over the upstream
+> OpenTelemetry Collector `loadbalancing` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.loadbalancing` components can be specified by giving them
+different labels.
+
+Note that either a trace ID or a service name is used for the decision on which backend to use. 
+The actual backend load isn't taken into consideration. Even though this load-balancer won't do 
+round-robin balancing of the batches, the load distribution should be very similar among backends 
+with a standard deviation under 5% at the current configuration.
+
+`otelcol.exporter.loadbalancing` is especially useful for backends configured with tail-based samplers
+which make a decision based on the view of the full trace.
+
+When a list of backends is updated, around `1/n` of the space will be changed, so that the same trace ID 
+might be directed to a different backend, where `n` is the number of backends. This should be stable enough 
+for most cases, and the higher the number of backends, the less disruption it should cause.
+
+## Usage
+
+```river
+otelcol.exporter.loadbalancing "LABEL" {
+  resolver {
+    ...
+  }
+  protocol {
+    otlp {
+      client {}
+    }
+  }
+}
+```
+
+## Arguments
+
+`otelcol.exporter.loadbalancing` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`routing_key` | `string` | Routing strategy for load balancing. | `"traceID"` | no
+
+The `routing_key` attribute determines how to route signals across endpoints. Its value could be one of the following:
+* `"service"`: exports spans based on their service name. This is useful when using processors like the span metrics, 
+so all spans for each service are sent to consistent collector instances for metric collection. 
+Otherwise, metrics for the same services would be sent to different collectors, making aggregations inaccurate.
+* `"traceID"`: exports spans based on their traceID.
+
+## Blocks
+
+The following blocks are supported inside the definition of
```

Comment:
```suggestion
The following blocks are supported in the definition of
```

Reply:
This is also a standard line which we put in many docs pages. If we are to change it, I think it should be done for all pages in a separate PR.

Reply:
@ptodev No problem. Leave it as-is then. Better we opt for consistency in the phrasing now and deal with the bigger issues :-)